ui/admin/app/styles/app.scss

```diff
@@ -735,6 +735,19 @@
   /* stylelint-enable selector-class-pattern */
 }
 
+.config-tags {
+  /* stylelint-disable selector-class-pattern */
+  .hds-table__tbody {
+    .hds-table__td {
+      &:last-child {
+        // remove width and white-space styles
+        width: unset !important;
```

Comment:
Is this `!important` needed? Really should avoid any usage of it and if we're using the hds classes (i.e. `hds-table__td` and `hds-table__tbody` we shouldn't be needing it unless the specificity is wrong.

Reply:
@ZedLi thats a great point, I think originally I had the specificity wrong and needed it.  Change incoming.